chsdi/views/wmtscapabilities.py

```diff
@@ -84,7 +84,7 @@ def wmtscapabilities(self):
         if hasattr(self.models['GetCapThemes'], 'oberthema_id'):
             themes = self.request.db.query(self.models['GetCapThemes']).order_by(self.models['GetCapThemes'].oberthema_id).all()
         else:
-            themes = self.request.db.query(self.models['GetCapThemes']).all()
+            themes = self.request.db.query(self.models['GetCapThemes']).all()  # pragma: no cover
```

Comment:
this is potentially dead code maybe better to just remove it if oberthema_id is always an attribute, please double check